src/ds5/ds5-options.cpp

```diff
@@ -657,7 +657,7 @@ namespace librealsense
     }
 
     auto_exposure_limit_option::auto_exposure_limit_option(hw_monitor& hwm, sensor_base* ep)
```

Comment:
The same change should be retrofitted to `    class auto_gain_limit_option : public option
` below

Reply:
fixed and tested